NuRadioReco/framework/event.py

```diff
@@ -21,6 +22,8 @@ def __init__(self, run_number, event_id):
         self.__radio_showers = collections.OrderedDict()
         self.__sim_showers = collections.OrderedDict()
         self.__event_time = 0
+        self.__particles = collections.OrderedDict()
+        self._generator_info = {}
```

Comment:
add a comment that describes what this dict contains.

Reply:
ok